interfaces/polkit/spec.go

```diff
@@ -59,6 +61,31 @@ func (spec *Specification) Policies() map[string]Policy {
 	return result
 }
 
+// AddRule adds a polkit rule file to install.
```

Comment:
should we do some validation on the nameSuffix even if probably the interface should also do some? even if we don't the comment here should probably say what is a valid nameSuffix

Reply:
unlike actions which need to be namespaced and can only contain the characters `[A-Z][a-z][0-9].-`, it seems that polkit rule file names don't have a specific pattern to follow.

https://www.freedesktop.org/software/polkit/docs/latest/polkit.8.html

Reply:
but this ends up in a filename so we need to have some opinion or do some escaping

Reply:
I added some heuristic validation, open to suggestion.